packages/website/components/table/table.js

```diff
@@ -157,41 +190,49 @@ function Table({
   if (isEmpty && !isLoading && emptyState) {
     return (
       <div className="storage-table">
-        <div className="storage-table-table-content">{emptyState}</div>
+        <div className="storage-table__content--loading">{emptyState}</div>
       </div>
     );
   }
 
   return (
     <div className="storage-table" role="table">
       <div role="rowgroup">
-        <div className="storage-table-row storage-table-header" role="row">
+        <div className="storage-table__row storage-table__header medium-up-only" role="row">
           {effectiveColumns.map(c => (
             <div key={c.id} role="columnheader">
               {c.headerContent}
             </div>
           ))}
         </div>
       </div>
-      <div className="storage-table-content">
+      <div className="storage-table__content">
         {isLoading ? <Loading className={'files-loading-spinner'} /> : rows.map((row, i) => renderRowComponent(row, i))}
 
         {!isEmpty && (
-          <div className="storage-table-footer">
+          <div className="storage-table__footer">
             <div>{!!leftFooterSlot && leftFooterSlot}</div>
-
-            <Pagination
-              className="storage-table-pagination"
+            <ServerPagination
+              className="files-manager-pagination"
+              itemsPerPage={rowsPerPage}
+              visiblePages={1}
+              pageCount={totalPages}
+              queryParam="page"
+              onChange={pageSelectHandler}
+              scrollTarget={'.account-files-manager'}
+            />
+            {/* <Pagination
```

Comment:
Shall we add some comments here to add some rationale behind why we kept using the "old" pagination?
It'd be great context for whoever picks this up in the future.
We could even be just a headline here and a link to the ticket to refactor with more context.

Reply:
Aye, added a TODO with some description to that comment block.